services/monitoring/docker-compose.yml.j2

```diff
@@ -274,8 +288,10 @@ services:
       resources:
         limits:
           memory: 256M
+          cpus: "0.5"
```

Comment:
I would have at least 1 CPU here for grafana.

Reply:
Done